bazel/external/quiche.genrule_cmd

```diff
@@ -52,8 +49,18 @@ cat <<EOF >sed_commands
 # Use envoy specific implementations for below platform APIs.
 /^#include/ s!"quiche_platform_impl/quiche_logging_impl.h!"source/common/quic/platform/quiche_logging_impl.h!
 /^#include/ s!"quiche_platform_impl/quiche_bug_tracker_impl.h!"source/common/quic/platform/quiche_bug_tracker_impl.h!
+/^#include/ s!"quiche_platform_impl/quiche_error_code_wrappers_impl.h!"source/common/quic/platform/quiche_error_code_wrappers_impl.h!
+/^#include/ s!"quiche_platform_impl/quiche_expect_bug_impl.h!"test/common/quic/platform/quiche_expect_bug_impl.h!
```

Comment:
can the quic_expect_bug_impl replacement above be removed?

Reply:
Oh, good point. I've removed this and also taken the opportunity to clean up a few other no longer useful lines.

Reply:
Oh, sure. Done